Dockerfile

```diff
@@ -1,11 +1,45 @@
-FROM cm2network/steamcmd:root
+FROM golang:1.22.0-alpine as rcon-cli_builder
+
+ARG RCON_VERSION="0.10.3"
+
+WORKDIR /build
+
+ENV CGO_ENABLED=0
+RUN wget -q https://github.com/gorcon/rcon-cli/archive/refs/tags/v${RCON_VERSION}.tar.gz -O rcon.tar.gz \
+    && tar -xzvf rcon.tar.gz \
+    && rm rcon.tar.gz \
+    && mv rcon-cli-${RCON_VERSION}/* ./ \
+    && rm -rf rcon-cli-${RCON_VERSION} \
+    && go build -v ./cmd/gorcon
+
+FROM cm2network/steamcmd:root as base-amd64
+# Ignoring --platform=arm64 as this is required for the multi-arch build to continue to work on amd64 hosts
+# hadolint ignore=DL3029
+FROM --platform=arm64 sonroyaalmerol/steamcmd-arm64:latest as base-arm64
```

Comment:
One thing you could do is use `sonroyaalmerol/steamcmd-arm64:root` instead of the `latest` tag. This would also allow you to remove `USER root` from the other linting error. While this is definitely not a fix, at least you'll have the linting errors suppressed for now.

Reply:
Oh hey, that's you!

What's the difference between the `root` and `latest` tags? I noticed that the `arm64` build used `latest`, so I was a bit leery of just swapping tags.

Though that would avoid the lint error, the "spirit" of the lint rule would still be violated, as it's point is to avoid tags that aren't deterministic

Reply:
The image is identical to `cm2network/steamcmd` (the base image used for amd64) but with the addition of Box64 and Box86 which adds compatibility for arm64. The `root` tag is literally just the `latest` tag but with `root` as its active user. The `latest` tag has `steam` as its active user as the steamcmd executable within the images are owned and can only be executed by `steam`.

The only reason the `Dockerfile.arm64` used the `latest` tag was that I hadn't implemented the `root` tag yet when I PR-ed the arm64 support.

Well, I can definitely add some deterministic tags for version pinning in my image but I can't say the same for the original `cm2network/steamcmd` image.

Reply:
Thanks for clarifying!